app/src/main/java/org/mozilla/fenix/library/history/HistoryAdapter.kt

```diff
@@ -101,11 +102,48 @@ class HistoryAdapter(
     var selected = listOf<HistoryItem>()
 
     fun updateData(items: List<HistoryItem>, mode: HistoryState.Mode) {
+        val diffUtil = DiffUtil.calculateDiff(
```

Comment:
Technically, this call should end up on a background thread. This was one of the findings of the performance audit. But it's okay if we make that change later.